docs/infra/ansible.md

```diff
@@ -0,0 +1,98 @@
+Ansible
+=======
+
+This page documents the Ansible configuration management repository at [FOSSRIT/infrastructure](https://github.com/FOSSRIT/infrastructure).
+Any changes to services or applications managed on FOSS@MAGIC infrastructure are managed in this git repo.
+
+
+## Contact information
+
+* **Owner**: FOSS@MAGIC Tech Team
+* **Contact**: `#rit-foss-admin` / `#rit-foss-admin:matrix.org` / [telegram.me/fossrit_infra](https://t.me/fossrit_infra)
+* **Persons**: [Justin W. Flory](https://github.com/jwflory), [Tim Zabel](https://github.com/tjzabel)
+* **Location**: Toronto, Canada (`TOR1`)
+* **Purpose**: Configuration management for FOSS@MAGIC infrastructure.
+
+
+## Background
+
+The FOSS@MAGIC Tech Team uses Ansible for system change management.
+Previously, no configuration management tool was used for our infrastructure.
+Some git commits go back to 2013 for a few scripts used back in the day, but the migration to Ansible started in September 2018.
+
+
+## Overview
+
+Ansible playbooks are run locally by a system administrator.
+A system administrator can run Ansible playbooks once a user and SSH key is set up for them on the host.
+ew SSH keys and users are defined in the `sshd` role (see [Roles](#roles)).
```

Comment:
`(see [Roles](#roles))` has a rendering issue. See screenshot

Reply:
@kennedy The screenshot was outdated from what is committed. I noticed the same mistake earlier. :sweat_smile: I originally had written `(see [Roles](#roles)`

Reply:
Oh i see.